src/modules/image-gen/utils/context.ts

```diff
@@ -0,0 +1,17 @@
+import { BotContext } from "../../types";
+import config from "../../../config";
+
+// on Menu ctx.from.id returns current user Id, on command returns bot id
+export const isAdmin = async (ctx: BotContext) => {
+  if (ctx.chat?.type === 'private') {
+    return true
+  }
+  const admins = config.appAdmins
+  const currentUser = ctx.from?.id || 0
+  const chatAdmins = await ctx.getChatAdministrators()
+  const adminsIds = chatAdmins.reduce<number[]>((result, item) => {
```

Comment:
seems like it can be simplified with map:
```
const adminsIds = chatAdmins.map(item => item.user.id).concat(admins)
```

Reply:
change made